### PR TITLE
Update manage-protection-updates-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated note to reflect that the Platform updates contain engine updates

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9707

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/manage-protection-updates-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ There are five locations where you can specify where an endpoint should obtain u
 To ensure the best level of protection, Microsoft Update allows for rapid releases, which means smaller downloads on a frequent basis. The Windows Server Update Service, Microsoft Endpoint Configuration Manager, Microsoft security intelligence updates, and platform updates sources deliver less frequent updates. Thus, the delta can be larger, resulting in larger downloads.
 
 > [!NOTE]
-> Security intelligence updates contain engine updates and are released on a monthly cadence.
+> Platform Updates updates contain engine updates and are released on a monthly cadence.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/manage-protection-updates-microsoft-defender-antivirus.md]:
```suggestion
> Platform updates contain engine updates and are released on a monthly cadence.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@yogkumgit The validation has been passed.

@denisebmsft Please merge the PR

**Comment by yogkumgit** [comment]
@denisebmsft Gentle reminder

**Pull request merged**